content/coop.html

```diff
@@ -128,23 +128,23 @@ <h3> Salary </h3>
 
 <div class="vbx2"></div>
 
-<h3> Coop Applications Breakdown</h3>
+<h3> Co-op Applications Breakdown</h3>
 <p class="full">
   <span class="big">
-  As respondents gained coop experience, they applied to less jobs over time
+  As respondents gained work experience, they applied to less jobs over time
   and found more jobs outside of Waterlooworks.
```

Comment:
```suggestion
  and found more jobs outside of WaterlooWorks.
```

(since i saw it here, i'll add the suggestion commit) apparently WaterlooWorks has a cap on the "Works" part. TIL.